crates/ruff/src/args.rs

```diff
@@ -515,101 +554,184 @@ impl From<&LogLevelArgs> for LogLevel {
     }
 }
 
+/// Struct to encapsulate all configuration-related arguments passed via the CLI.
```

Comment:
Nit: I wouldn't include the "Struct to encapsulate" since that's kind of redundant